docroot/themes/custom/uids_base/scss/admin.scss

```diff
@@ -1605,6 +1605,11 @@ only screen and (min-resolution: 1.5dppx) {
   border-color: transparent;
 }
 
+// Edit layout for textblock not displaying correctly
+.block-inline-blockuiowa-text-area {
+  min-height: 10px;
```

Comment:
Could we increase the min-height a bit? The configure button is reachable, but can sometimes overlap other parts where it gets less clear what it's attached to.

![image](https://user-images.githubusercontent.com/49074566/122819297-e34c6600-d29f-11eb-944e-4e32b7818522.png)

~40px gives enough space to fully contain the button. Not sure if that's so much space that might make the preview a bit misleading for what is actually in the text block, though.

![image](https://user-images.githubusercontent.com/49074566/122819589-49d18400-d2a0-11eb-9a56-b0b8153b10d8.png)

Reply:
Woof! Thank you! IMO the 40px works great!